contact.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar:
+nav: "docs"
+permalink: /contact.html
+layout: splash
+header:  
+    video: /assets/images/waves.mp4
+    overlay_color: "rgba(63, 108, 152, 1)"
+intro:
+  - excerpt: "Contact"
+---
+
+{% include feature_row id="intro" type="center" %}
+<div class="feature__wrapper">
+  <div id="contact" class="content-container text-center"> 
+      <div style="width: 250px; margin: 0 auto 30px">
```

Comment:
Avoid the inserting of styles inside the HTML, that's what the CSS files are for.